packages/flutter/test/widgets/image_test.dart

```diff
@@ -1981,3 +2027,36 @@ class FailingImageProvider extends ImageProvider<int> {
     );
   }
 }
+
+class TestImageStreamCompleter2 extends ImageStreamCompleter {
```

Comment:
nit: Can you pick a more descriptive name here? 

Alternatively, can we just merge this class with TestImageStreamCompleter in here?

Reply:
Yes, I think it is possible to merge.

Reply:
@dnfield I think it is better now.